src/yocto/recipes.md

```diff
@@ -0,0 +1,186 @@
+# Recipes
+Official page: https://docs.yoctoproject.org/overview-manual/concepts.html#recipies
+
+## Recipes files
+Recipe files or also known as BB files, are files that contain information and instructions that build tool (bitbake) takes for generating the packages. This information for instructions can be resumed as:
```

Comment:
"This information for instructions can be resumed as:" is not concise, shorten it